_posts/blog/2018-11-29/2018-11-29-openbazaar-2-turns-1-lessons-learned.markdown

```diff
@@ -0,0 +1,46 @@
+---
+title: "OpenBazaar 2 Turns 1: <span class='post-title-extra'>Lessons Learned</span>"
+layout: post
+date: '2018-11-29 06:30:00 -0300'
+social_title: 'OpenBazaar 2 Turns 1: Lessons Learned'
+social_description: 'OpenBazaar started more than four years ago, and for everyone involved it has been a learning experience. Here are a few lessons we’ve learned over the years, with a particular emphasis on the past year.'
+---
+
+This month we’ve been celebrating [the one year anniversary of OpenBazaar 2.0](https://openbazaar.org/blog/these-5-openbazaar-stats-are-really-exciting/). To read a summary of the last year, read [Brian Hoffman’s piece](https://openbazaar.org/blog/big-change-hard-work-and-strong-growth-openbazaar-2-turns-1/).
+
+OpenBazaar started more than four years ago, and for everyone involved it has been an incredible learning experience. Here are a few lessons we’ve learned over the years, with a particular emphasis on the past year.
+
+#### 1. Expect politics.
+
+Just like Bitcoin, OpenBazaar is an open source piece of software and network of people running the software. As such, it’s incapable of having a “position” on any given issue. However, the individuals and companies building OpenBazaar do have opinions about issues in the cryptocurrency space. When they promote these beliefs – particularly on controversial issues – others will often claim “OpenBazaar believes X” instead of correctly attributing the opinion to the individual or company.
```

Comment:
> OpenBazaar is an open source piece of software and network of people

I think you need a "a" in front of network since "an" can't be used for both open and network.

ie: "OpenBazaar is an open source piece of software and a network of people"